olgabot/2022 jun comments (#12)
dib-lab/2021-paper-metapangenomes: content/03.results.md

**D)** Pangenome metrics strongly correlate between gene- and k-mer-based pangenomes. Pangenome categories core, shell, and cloud refer to genes or k-mers shared between the majority (>95%), some, or singleton genomes in the pangenome. Alpha is a value from Heaps law used to estimate whether a pangenome is open or closed.
](images/panmers_fig.png){#fig:panmers_fig}

<!-- Olga started here  -->
We next investigated whether other pangenome metrics were well correlated between our k-mer-based method and the gene-based method roary (see Methods for details).
For 13 pangenomes, the percent of k-mers or genes predicted to be part of the core, shell, or cloud pangenome was strongly correlated (**Figure @fig:panmers_fig** D).

> The content of the core genomes as computed was also similar between pangenomes built with different methods.

Reviewer: ```suggestion
The content of the core genomes was also similar between pangenomes built with different methods.
```

> <!-- How was the content of core genomes assessed? -->

Reviewer: ```suggestion
```

Focusing on genes or k-mers shared between all genomes in a pangenome, and limiting our inquiry to pangenomes with at least five genes shared between all genomes (n = 11), we found core k-mers contained an average of 83.9% (SD = 15.4%) of sequences in core genes, while core genes contained an average of 73.5% (SD = 16.9%) of sequences in core k-mers.

> <!-- Which figure should I be looking at for the 83.9% and 73.5% values? -->

Reviewer: ```suggestion
```

For reads simulated from genomes not in GTDB, orpheum recovered the majority of coding reads when genomes of the same species were in the database (**Figure @fig:orpheum_fig A,B**).
On average, 30.2% (SD = 27.1%) of reads that were coding were predicted to be non-coding, while 4.8% (SD = 5.5%) of reads that were non-coding were predicted to be coding.
Accuracy decreased with increasing taxonomic distance between the query genome and the closest relative in the database (**Figure @fig:orpheum_fig B**).

> <!-- Is it possible to compute distance by mya from GTDB-tk? -->

Reviewer: ```suggestion
```

For genomes that had at least species-level representatives in GTDB, the largest source of error was non-coding reads being predicted as coding (**Figure @fig:orpheum_fig A**).
We hypothesized that these reads originated from pseudogenes as these sequences would likely not be annotated as coding in the genomes from which the reads were simulated from, but may retain some k-mers contained in the database.
To assess this hypothesis, we used annotation files produced by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP), which annotates pseudogenes, for the 23 genomes for which these files were available [@doi:10.1093/nar/gkw569; @doi:10.1093/nar/gkaa1105].
On average, 12.4% (SD = 13.8%) of non-coding reads that were predicted to be coding fell within pseudogenes annotated by the PGAP pipeline.

> <!-- Is there a figure for noncoding reads in pseudogenes? -->

Reviewer: ```suggestion
```

We hypothesized that this was caused by sequencing error introduced into the simulated reads.
We mapped the simulated reads against the coding domain sequences from which they were derived and calculated mapping error rates. 
While all reads mapped, the error rate was higher for reads that were predicted to be non-coding than those predicted to be coding (Welch Two Sample t-test, estimate = 0.00523, p < 0.001).

> <!-- Is there a figure for read error rates in coding vs noncoding? -->

Reviewer: ```suggestion
```

Decreasing the Jaccard containment threshold increased the sensitivity and specificity of ORF prediction when there are no closely related genomes in the database (**Figure @fig:orpheum_fig C, Table @tbl:threshold**).
The Jaccard containment threshold controls the final prediction of coding vs. non-coding, as well as the the number of open reading frames which a read is translated into.

> <!-- Is "genome" the preferred name of taxonomic rank over "strain" in GTDB? -->

Reviewer: ```suggestion
```

On average, increasing the rank of the closest taxonomic relative in the database by one taxonomic level decreased the optimal Jaccard containment threshold by 0.13.

> We note that orpheum has performs well, achieving sensitivity > 0.9 or better, when species of the same genome, species, and genus are present, but decreases significantly when the next closest relative is at the family level (**Figure @fig:orpheum_fig C).

Reviewer: ```suggestion
We note that orpheum performed well, achieving sensitivity > 0.9 or better, when genomes of the same strain, species, and genus are present, but decreases significantly when the next closest relative is at the family level (**Figure @fig:orpheum_fig C).
```

We used a k-mer size of 51, as this is indicative of strain-level similarity [@doi:10.1128/mSystems.00020-16] (CITE: tessa).
In each of the three species we investigated, we identified different patterns of strain fluctuations.

> In *Bacteroides uniformis*, only one strain (BIOML-A27) was present until week 25, when another strain appears (BIOML-A2).

Reviewer: ```suggestion
In *Bacteroides uniformis*, only one strain of *B. uniformis* (BIOML-A27) was present until week 25, when another strain appears (BIOML-A2).
```

> <!-- What does "other" mean In figure 4D? The text states that BIOML-A27 was the only strain of *B. uniformis*, but it seems like that is not true "other" is present -->

Reviewer: ```suggestion
```
